e2e/api_test.go

```diff
@@ -73,16 +74,20 @@ func TestAPI(t *testing.T) {
 	}
 
 	cfg, err := config.New()
-	if err != nil {
-		panic(err)
-	}
+	require.NoError(t, err)
+
 	cosmos.CustomizeConfig(cfg)
 
 	conn, err := grpc.DialContext(context.Background(), "localhost:50052", grpc.WithInsecure())
 	require.NoError(t, err)
 
 	kb, err := cosmos.NewKeybase(filepath.Join(cfg.Path, cfg.Cosmos.RelativePath))
```

Comment:
the ci error is still the same:
```
couldn't create db: Error initializing DB: stat /home/circleci/project/e2e.test/mesg/cosmos/keys/keys.db: permission denied
```
simply use another folder. eg:
```suggestion
	kb, err := cosmos.NewKeybase(filepath.Join(cfg.Path, cfg.Cosmos.RelativePath, 'e2e-keybase'))
```